src/KeyboardDatePicker.tsx

```diff
@@ -1,69 +1,11 @@
-import React from 'react';
+import { DatePicker, DatePickerProps } from './DatePicker';
 
-import {
-	KeyboardDatePicker as MuiKeyboardDatePicker,
-	KeyboardDatePickerProps as MuiKeyboardDatePickerProps,
-} from '@material-ui/pickers';
+/**
+ * @deprecated use DatePicker instead
+ */
+export const KeyboardDatePicker = DatePicker;
```

Comment:
What about even wrapping with a console.log?

Reply:
Ok

Reply:
Also, should there be Desktop(Date|Time|DateTime)Picker and Mobile(Date|Time|DateTime)Picker ? If so, should I just copy the existing pickers and tests?

Reply:
We only need to keep the existing API. No need to add additional files.